### PR TITLE
Now analyzing mac files, skipping note tho

### PR DESCRIPTION
**Pull request by yelenderosen**

### CODE REVIEW

File: edr_connect/proxies/cs_edr_proxy.py [hunk 1]
```diff
@@ -131,14 +131,11 @@ def fetch_latest_alerts(self, hours: int, handled_alerts: Set[str]) -> List[Aler
                 incident_id_by_detection_id[incident_detection_behavior['detection_ids'][i]] = incident_id
 
         detections = self.detect.query_detects(filter=f"created_timestamp:>='{hours_ago}'",
-                                               sort='date_updated|desc',
+                                               sort='last_behavior|desc',
                                                limit='9999')
         _logger.debug(f"Found {len(detections['body']['resources'])} alerts in the EDR in the last {hours} hours")
 
         for detection_id in detections['body']['resources']:
-            if detection_id in handled_alerts:
```
[the remark this answers is not in the document]
Comment (yelenderosen) [reply edr_connect/proxies/cs_edr_proxy.py:139]: it doesn't work...

File: edr_connect/managers/analysis_manager.py [hunk 2]
```diff
@@ -189,11 +185,20 @@ def send_notes(self, file_hash: str, analysis: Analysis):
...
+        self._filter_out_mac_alerts(alerts)
+        if alerts:
+            alerts_id = self.edr_proxy.send_note(alerts, analysis)
+            _logger.info(f'Note sent for alerts {", ".join(alerts_id)}')
         self.handled_alerts.update(alert.alert_id for alert in alerts)
         del self.running_analysis_id_and_alerts_by_hash[file_hash]
 
+    @staticmethod
+    def _filter_out_mac_alerts(alerts: Set[AlertInfo]):
+        for alert in alerts.copy():
```
[the remark this answers is not in the document]
Comment (yelenderosen) [reply edr_connect/managers/analysis_manager.py:197]: why do we need the copy?
python hates when you change a list you iterate on

[the remark this answers is not in the document]
Comment (davidt99) [reply edr_connect/managers/analysis_manager.py:197]: i meaning why we need to remove the alert from the list and not just skip it

File: edr_connect/proxies/cs_edr_proxy.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (yelenderosen) [reply edr_connect/proxies/cs_edr_proxy.py:139]: I added that and then figured out that the key in handled alerts is composed from detection_id + behavior_id..
So i'm removing it now

**Review by davidt99 — APPROVED** [review]

**Pull request merged**